libs/datamodel/connectors/dml/src/native_type_instance.rs

```diff
@@ -32,3 +38,24 @@ impl NativeTypeInstance {
         serde_json::from_value(self.serialized_native_type.clone()).expect(&error_msg)
     }
 }
+
+impl TryFrom<NativeTypeInstance> for MsSqlType {
+    type Error = native_types::Error;
+
+    fn try_from(value: NativeTypeInstance) -> Result<Self, Self::Error> {
+        let args = value
+            .args
+            .iter()
+            .map(|v| format!("{}", v))
+            .collect::<Vec<_>>()
+            .join(",");
+
+        let typ = if args.len() > 0 {
+            format!("{}({})", value.name, args).parse()?
```

Comment:
Is this the prisma schema name or the database name, or are they the same?

Reply:
Yes.